source/less/blocks/reviews.less

```
@@ -2,13 +2,44 @@
   padding: 58px 30px 45px 30px;
 }
 
+@media (min-width: @tablet-width) {
+  .reviews {
+    padding: 79px 50px 68px 50px;
+    background-image: url("../../img/review-tablet.jpg");
```

```suggestion
    background-image: url("../img/review-tablet.jpg");
```
Иначе вы указываете браузеру 2 перехода вверх, а это уже за пределами корня сайта.